auth/src/main/java/com/vimeo/networking2/Authenticator.kt

```diff
@@ -56,7 +58,11 @@ import okhttp3.Credentials
  * Authenticator.initialize(Configuration)
  * val instance = Authenticator.instance()
  * ```
+ *
+ * See [https://developer.vimeo.com/api/authentication] for more information about the different authentication
+ * approaches.
```

Comment:
Instead of linking to the `MainActivity` which is a bit awkward since the example isn't packaged with the library, I added more to the `Authenticator` documentation block about client credentials vs basic authentication vs code grant. We will be linking to the example in the readme as well.